<launch>
  <!-- 开启机器人底层相关节点  -->
  <include file="$(find wheeltec_arm_pick)/launch/base_serial.launch" />
  <!-- 开启RGB摄像头  -->
  <include file="$(find usb_cam)/launch/usb_cam-test.launch" />
  <!-- 开启深度摄像头  -->
  <include file="$(find astra_camera)/launch/astra.launch" />
  <!-- 开启色块识别程序  -->
  <include file='$(find wheeltec_arm_pick)/launch/visualTracker.launch' />
  <!-- 开启虚拟机械臂  -->
  <include file='$(find wheeltec_arm_moveit_config)/launch/demo.launch' />
  <!-- 开启测试参数节点  -->  
  <node name='test_param' pkg="wheeltec_arm_pick" type="test_param" output="screen"/>

</launch>
